perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -481,6 +481,13 @@ sub munge_report_new_contacts {
     } else {
         @$categories = grep { grep { $_ ne 'Waste' } @{$_->groups} } @$categories;
     }
+
+    my $user = $self->{c}->user;
+    if ($user && $user->from_body($self->body) && $user->get_extra_metadata('assigned_categories_only')) {
+        my %user_categories = map { $_ => 1} @{$user->categories};
+        @$categories = grep { $user_categories{$_->category} } @$categories;
```

Comment:
Does this need the additional Bromley subcategory stuff, or has that already been dealt with? (e.g. like elsewhere in this file that also looks at `$c->user->get_extra_metadata('subcategories')`).

Reply:
We agreed with Bromley that we were going to ignore the subcategory stuff for this piece of work because we're going to be turning them into "normal" subcategories as part of the next piece of work for them.